src/main/resources/admin.vm

```diff
@@ -460,6 +460,15 @@
     </div>
    </fieldset>
 
+  <fieldset class="group">
+   <legend>
+    <span>Force Merge on Rescope</span>
```

Comment:
Perhaps it can be changed to "Update Pull Request References".

And add a ```
<div class="description">Will update refs/pull-request/{id}/from and refs/pull-request/{id}/merge references in git.</div>
```

Reply:
:+1: That wording is much better than mine :) Will update.